### PR TITLE
Fixes roundstart real_name DNA, and mitocholide rejuv

### LINKED ISSUE #3968: People's True DNA isn't correctly assigned
Cloned people and genetics machines are giving random names.

### PR DESCRIPTION
**Pull request by Crazylemon64**
Fixes #3968

:cl:Crazylemon
bugfix: You now will actually have the correct `real_name` on your DNA.
bugfix: No more roundstart runtimes regarding IPC hair.
bugfix: Mitocholide rejuv will work more usefully now.
/:cl:

### CODE REVIEW

**Comment by fludd12** [comment]
Speaking of IPC stuff, can you make the unbranded head use monitor hairs? It looks basically the same as a normal head.

File: code/modules/surgery/organs/organ_external.dm [hunk 1]
```diff
@@ -320,6 +321,8 @@ This function completely restores a damaged organ to perfect condition.
 	for(var/obj/item/organ/internal/current_organ in internal_organs)
 		current_organ.rejuvenate()
 
+	for(var/obj/item/organ/external/EO in contents)
```
Comment (tigercat2000) [inline code/modules/surgery/organs/organ_external.dm]: ... do external organs even have external organs in them? Wat.

Comment (Fox-McCloud) [inline code/modules/surgery/organs/organ_external.dm]: Chop off a left leg--it contains the left foot.

This was a problem we ran into last night; even when you reset the germ level and poured mito on an external limb, then re-attached it, the child limb would still be "dead".

File: code/modules/mob/living/carbon/human/appearance.dm [hunk 2]
```diff
@@ -184,14 +178,17 @@
 		if(gender == FEMALE && S.gender == MALE)
 			continue
 		if(species.flags & ALL_RPARTS) //If the user is a species who can have a robotic head...
+			var/obj/item/organ/external/head/H = organs_by_name["head"]
+			if(!H)
+				return
 			if(species.name in S.species_allowed) //If this is a hairstyle native to the user's species...
-				if(!user.client.prefs.rlimb_data["head"]) //Check to see if they have the default head.
+				if(H.model == "Morpheus Cyberkinetics") //Check to see if they have the default head.
```
Comment (KasparoVy) [inline code/modules/mob/living/carbon/human/appearance.dm]: :hankey:, it was actually that simple?

HHHHHNNNGG

**Pull request merged**